source/common/util/generate-filename.js

```diff
@@ -23,5 +25,9 @@ module.exports = function () {
   let pattern = global.config.get('newFileNamePattern')
   pattern = replaceStringVariables(pattern)
   pattern = pattern.replace(/%id/g, generateId(global.config.get('zkn.idGen')))
+  // In case a funny guy has removed the pattern from config.
+  if (pattern.length === '') pattern = uuid()
```

Comment:
```suggestion
  if (pattern.length == 0) pattern = uuid()
```

Reply:
I must've been asleep while coding that … thanks!